src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx

```diff
@@ -27,7 +30,7 @@ export default class PhoneNumberWidget extends React.Component {
     return (
       <TextWidget
         {...this.props}
-        inputType={'tel'}
+        options={{ inputType: 'tel' }}
```

Comment:
this never worked :( 

https://github.com/department-of-veterans-affairs/vets-website/blob/150a9867939b011ab385b908fa54cae67dd12ec0/src/platform/forms-system/src/js/widgets/TextWidget.jsx#L7